### DISCUSSION TITLE
How to hook into NextAuth using ConnectKit

### DISCUSSION THREAD

**Discussion by launchthatbrand** [family/connectkit#135] [Q&A]
How can I integrate connectkit and NextAuth?

There is an example here:
https://blog.spruceid.com/sign-in-with-ethereum-on-next-js-applications/

and other examples online which a custom login button which fires:
``await connect(connectData.connectors[0]);`

followed by the NextAuth signIn function:
`signIn('credentials', { message: JSON.stringify(message), redirect: false, signature, callbackUrl });`

When using Wagmi with multiple providers and connectkit the buttons are generated. How do i hook into things to fire the NextAuth signIn function?

**Comment by dylanngph** [comment]
Yeah, no support yet, pump this up
